### ISSUE TITLE
`allow` value to allow all licenses?

### ISSUE THREAD

**Issue by silverwind** [microsoft/license-checker-webpack-plugin#33]
Is there a value for `allow` that would accept all licenses? I use this module to generate a text file and I don't really care about the actual licenses or their verification.

**Comment by unindented** [comment]
This package supports any valid SPDX expression. It uses [`spdx-satisfies`](https://www.npmjs.com/package/spdx-satisfies) under the hood. If you find an expression that works with `spdx-satisfies`, it'll work here too.

**Comment by silverwind** [comment]
Yeah I search already and my impression is that the SPDX syntax has no such expression, it only supports explicitely named licenses with combinators.

**Comment by unindented** [comment]
@silverwind I'm open to accepting a PR that adds this feature.

**Comment by silverwind** [comment]
Will take a look later. Do you have a preference for either "all" or "any"?

**Comment by unindented** [comment]
Let's use `any`.
